lib/job-iteration/iteration.rb

```diff
@@ -178,6 +178,8 @@ def interruptible_perform(*arguments)
           instrumentation_tags.merge(times_interrupted: times_interrupted, total_time: total_time),
         )
       end
+
+      nil
```

Comment:
I put this here because `perform` just delegates to this method, but maybe it would make more sense to put it there?


```diff
 def perform(*params) # @private
   interruptible_perform(*params)
+  
+  nil
 end
 
 def interruptible_perform(*arguments)
   # ...
-  
-  nil
 end
```

Not sure...

Reply:
We're making this change because we care about the return value of `perform`, so it makes sense to me to put it there, instead of here. Not a big deal, though 🤷

Reply:
Yeah, that makes sense. I've updated it.